DropwizardHelloWorld/src/main/java/com/example/helloworld/HelloWorldResource.java

```diff
@@ -0,0 +1,38 @@
+package com.example.helloworld;
+
+import com.example.helloworld.Saying;
+import com.codahale.metrics.annotation.Timed;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.Optional;
+
+@Path("/hello-world")
+@Produces(MediaType.APPLICATION_JSON)
```

Comment:
Please, put the annotations just above the class definition. It would read better.